docs/source/getting_started/dataset_creation/dataset_creation_aws_bucket.rst

```diff
@@ -87,6 +87,64 @@ Let us assume your bucket is called `datalake`. And let us assume the folder you
 
         Get security credentials (access key id, secret access key) from AWS
 
+
+
+
+**S3 IAM Delegated Access**
+
+To access your data in your S3 bucket on AWS, Lightly `can assume a role <https://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html>`_ in your account which has the necessary permissions to access your data.
+This is `considered best practice <https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#delegate-using-roles>`_ by AWS.
+
+To set up IAM Delegated Access
+
+1. Go to the `AWS IAM Console <https://console.aws.amazon.com/iam/home?#/roles>`_
+
+2. Click `Create role`
+   
+3. Select `AWS Account` as the trusted entity type
+
+    a. Select `Another AWS account` and specify the AWS Account ID of Lightly: `311530292373`
+
+    b. Check `Require external ID`, and choose an external ID. The external ID should be treated like a passphrase
+
+    c. Do not check `Require MFA`.
+    
+    d. Click next
+
+4. Select a policy which grants access to your S3 bucket. If no policy has previously been created, here is an example of how the policy should look like:
+
+
+
+    .. code-block:: json
+            
+        {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Sid": "lightlyS3Access",
+                    "Action": [
+                        "s3:*Object",
+                        "s3:ListBucket"
+                    ],
+                    "Effect": "Allow",
+                    "Resource": [
+                        "arn:aws:s3:::{YOUR_BUCKET}/*",
+                        "arn:aws:s3:::{YOUR_BUCKET}"
+                    ]
+                }
+            ]
+        }
+
+5. Name the role `Lightly-S3-Integration` and create the role.
+6. Edit your new `Lightly-S3-Integration` role: set the `Maximum session duration` to 12 hours. 
+
+    .. warning:: If you don't set the maximum duration to 12 hours, Lightly will not be able to access your data. Please make sure to se the `Maximum session duration` to 12 hours.
+
+
+7. Remember the external ID and the ARN of the newly created role
```

Comment:
ID should be `367053757506 ` and ARN `arn:aws:iam::367053757506:role/Lightly-S3-Integration` right? If yes then I would write it directly in the docs.

Reply:
Yes, makes sense 👍

Reply:
(the ID is not `367053757506` but more of a password).

Reply:
> (the ID is not 367053757506 but more of a password).

What is the ID then?